packages/dso-toolkit/src/components/card/card.args.ts

```diff
@@ -151,6 +129,6 @@ export function cardArgsMapper<TemplateFnReturnType>(
         }
       : undefined,
     content,
-    dsoCardClicked: (e) => a.dsoCardClicked?.(e.detail),
+    dsoCardClick: (e) => a.dsoCardClick?.(e.detail),
```

Comment:
Dan is deze optional chain operator ook niet meer nodig

Reply:
Is weg

Reply:
Gone like the wind